### PR TITLE
Issue 32 rescale

### PR DESCRIPTION
**Pull request by eric-bueno**
Made progress on rescale function, see commit notes for details

### CODE REVIEW

**Comment by eric-bueno** [comment]
I also created a function which decomposes mask composites. It turns out it is unnecessary at the moment; perhaps we will find a use for it later

File: functions/rescaling.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    """
+    curr_size = compute_avg_size(composite)
+    ratio = target_size / curr_size
+    ratio = ratio ** (1/2)
+    new_shape = (int(composite.shape[0] * ratio),
+                 int(composite.shape[1] * ratio))
+    
+    rescaled_comp = rescale(composite, ratio, anti_aliasing=False,
+                            preserve_range=True, order=0)
+    resized_img = resize(image, new_shape, anti_aliasing=True,
```
Comment (jccaicedo) [inline functions/rescaling.py]: Any reason why we use `rescale` for composites and `resize` for images? Can we use the same for both?

**Review by jccaicedo — COMMENTED** [review]
Oh! Interesting. I thought we had to do rescaling one mask at a time. I'm glad it is not required. Looking forward to see some results.

**Review by jccaicedo — APPROVED** [review]
I saw that rescaling one mask at a time was not necessary. That's very interesting.
My only question is about the choice of `rescale` vs `resize` for images and composites. Please, add a comment with the reason for this choice. But no further action required, you can merge as is.

**Comment by eric-bueno** [comment]
I made the use of resizing consistent in rescale_composite()

Initially I had used rescaling and resizing interchangeably because the shapes of masks and images made one function more convenient than the other. This has been fixed in the most recent commit

**Pull request merged**